src/neo4j_graphrag/experimental/components/lexical_graph.py

```diff
@@ -103,19 +108,23 @@ def create_document_node(self, document_info: DocumentInfo) -> Neo4jNode:
         """
         document_metadata = document_info.metadata or {}
         return Neo4jNode(
-            id=document_info.path,
+            id=document_info.document_id,
             label=self.config.document_node_label,
             properties={
                 "path": document_info.path,
+                "createdAt": str(datetime.now()),
```

Comment:
I think this should be timezone aware. Maybe:

```python
"createdAt": datetime.now(timezone.utc).isoformat(),
```

Reply:
Updated in https://github.com/neo4j/neo4j-graphrag-python/pull/222/commits/96451c5830cc308cc3bb2306e7bb7a39ac9513b7 (same syntax as RunResult that prevents the deprecation warning and works for all supported Python versions).

Nice suggestion of the `isoformat` method btw :)